scripts/playerLogGenerator.py

```diff
@@ -0,0 +1,70 @@
+import json
+from faker import Faker
+from enum import Enum
+import random
+
+# Initialize Faker
+fake = Faker()
+
+Intention = Enum('Intention', ['move', 'follow_ball', 'get_ball', 'dribble', 'shield_ball', 'pass', 'shoot_on_target', 'keeperMove', 'self_pass'])
+def generate_robot():
+    return {
+        "id": random.randint(1, 7),
+        "pose": [random.uniform(-7.000, 7.000), random.uniform(-11.000, 11.000), random.uniform(-1.000, 5.000)],
```

Comment:
Is this the position (wm_self_pos_x, wm_self_pos_y, wm_self_pos_r)?

Reply:
all parameter names are taken from the MSL-provided example logs. will discuss with niels if changes are needed